packit/upstream.py

```diff
@@ -356,18 +357,24 @@ def sync_files(self, downstream_project: LocalProject) -> None:
 
     def get_version(self) -> str:
         """
-        Return version of the latest release available: prioritize upstream
-        package repositories over the version in spec
+        Return version of the latest release available: prioritize bigger from upstream
+        package repositories or the version in spec
         """
-        ups_ver = self.get_latest_released_version()
-        spec_ver = self.get_specfile_version()
-        # we're running both so that results of each function are logged and user is aware
-        if ups_ver:
+        ups_ver = version.parse(self.get_latest_released_version() or '')
+        logger.debug(f"Version in upstream package repositories: {str(ups_ver)}")
```

Comment:
I'd say that `{}` implicitly calls str(), but I can be wrong.

Reply:
You're right. I'll update it.